ui/components/component-library/checkbox/checkbox.stories.tsx

```diff
@@ -19,186 +24,154 @@ export default {
     label: {
       control: 'text',
     },
+    name: {
+      control: 'text',
+    },
+    id: {
+      control: 'text',
+    },
   },
-  args: {},
 } as Meta<typeof Checkbox>;
 
 const Template: StoryFn<typeof Checkbox> = (args) => {
-  const [isChecked, setIsChecked] = React.useState(false);
-
+  const [{ isChecked }, updateArgs] = useArgs();
```

Comment:
useArgs hook allows for you to change controls so we can now change the state via control or interaction with the component and vice versa


https://github.com/MetaMask/metamask-extension/assets/8112138/dab8b8f6-8c0c-4c48-bb12-c34bd9cb63be